.pre-commit-config.yaml

```diff
@@ -0,0 +1,35 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+      - id: trailing-whitespace
+        args: [--markdown-linebreak-ext=md]
+      - id: end-of-file-fixer
+      - id: check-json
+      - id: check-toml
+      - id: check-yaml
+      - id: pretty-format-json
+        exclude: package(-lock)?\.json
+        args:
+          - --autofix
+      - id: requirements-txt-fixer
```

Comment:
can we also add `double-quote-string-fixer` to that list? ... basically I just prefer single quotes for strings in general :smiley: